pom.xml

```diff
@@ -27,7 +27,7 @@ The MIT License (MIT)
   <parent>
     <groupId>com.jcabi</groupId>
     <artifactId>parent</artifactId>
-    <version>0.50.0</version>
+    <version>0.50.5</version>
```

Comment:
@victornoel Why `0.50.5` if `0.50.7` is the latest?

Reply:
@andreoss good point, but since it is marked as pre-release on the github release page, I preferred to use the version meant to be stable and also I know it works as I expect.

Reply:
@victornoel Okay. Looks good than